### ISSUE TITLE
Add `zod-accelerator` to zod ecosystem.

### ISSUE THREAD

**Issue by mathcovax** [colinhacks/zod#3342]
I want to congratulate you for creating zod. There is for me, no better library than yours in the field of parsing!

zod’s two strengths are its statement and typing. Unfortunately, it is fishing for performance. 😬

That’s why I created a library that solves this problem. Like that, no more excuses for not wanting to use zod (server side). 😁

repo : https://github.com/duplojs/duplojs-zod-accelerator

I thank you again for the work you have done 🫡

**Comment by telemakhos** [comment]
Maybe you can just add the link to the readme and send a PR?

**Comment by telemakhos** [comment]
There you go... https://github.com/colinhacks/zod/pull/3388

**Comment by mathcovax** [comment]
thank you for doing it 😄

**Comment by telemakhos** [comment]
You're welcome... I'd just try to remove the dependencies and have it as a package with 0 dependencies if you want to see a big adoption. I think it could get very popular or even be merged into zod itself.

**Comment by jraoult** [comment]
@mathcovax I just found your "flavour" of Zod. I might have missed the relevant section explaining it already, but the gains are so impressive that I wonder why it is faster and what the trade-offs are.

**Comment by mathcovax** [comment]
> @mathcovax I just found your "flavour" of Zod. I might have missed the relevant section explaining it already, but the gains are so impressive that I wonder why it is faster and what the trade-offs are.

Hi, unfortunately I don’t have enough time to write a section about it. However I can still answer you! there is my sense 2 losses, the first is the reference to the zod object, zod-accelerator performs a "print". So if you change something after using zod-accelerator, it will not affect the function returned by zod. The tenth disadvantage is the build time of a schema. if you build to run only once the schema, you will not be profitable.

**Comment by jraoult** [comment]
@mathcovax, thanks. In the meantime, I researched and realised that the `zod-accelerator` gains most of its speed from generating an ad-hoc validator function at runtime, which I didn't understand at first.
